dev-python/jupyterlab_widgets/jupyterlab_widgets-3.0.10.ebuild

```diff
@@ -0,0 +1,24 @@
+# Copyright 1999-2024 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+DISTUTILS_USE_PEP517=jupyter
+PYTHON_COMPAT=( pypy3 python3_{10..12} )
+PYTHON_REQ_USE="threads(+)"
+
+inherit distutils-r1 pypi
```

Comment:
What about using the github tarballs instead, they have tests. Would this work?

Reply:
I have copied and adapted the ebuild from widgetsnbextension